app/jobs/scheduled/auto_join_users.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Jobs
+  class AutoJoinUsers < ::Jobs::Scheduled
+    every 6.hours
```

Comment:
I wonder if we don't want this more often? Like 1 hour?

Reply:
Sounds good to me!